core/src/main/java/org/elasticsearch/index/fielddata/ScriptDocValues.java

```diff
@@ -253,124 +254,41 @@ public int size() {
             return values.count();
         }
 
-        public double factorDistance(double lat, double lon) {
-            GeoPoint point = getValue();
-            return GeoDistance.FACTOR.calculate(point.lat(), point.lon(), lat, lon, DistanceUnit.DEFAULT);
-        }
-
-        public double factorDistanceWithDefault(double lat, double lon, double defaultValue) {
-            if (isEmpty()) {
-                return defaultValue;
-            }
-            GeoPoint point = getValue();
-            return GeoDistance.FACTOR.calculate(point.lat(), point.lon(), lat, lon, DistanceUnit.DEFAULT);
-        }
-
-        public double factorDistance02(double lat, double lon) {
-            GeoPoint point = getValue();
-            return GeoDistance.FACTOR.calculate(point.lat(), point.lon(), lat, lon, DistanceUnit.DEFAULT) + 1;
-        }
-
-        public double factorDistance13(double lat, double lon) {
-            GeoPoint point = getValue();
-            return GeoDistance.FACTOR.calculate(point.lat(), point.lon(), lat, lon, DistanceUnit.DEFAULT) + 2;
-        }
-
         public double arcDistance(double lat, double lon) {
```

Comment:
we can. Is there a particular motivation for removing `doc['loc'].arcDistance(tgtLat, tgtLon)` in favor of `org.elasticsearch.common.geo.GeoUtils.arcDistance(doc['loc'].lat, doc['loc'].lon, tgtLat, tgtLon)`? Seems like the former is more intuitive? Missing values would also have to be explicitly handled in scripts if we remove the `withDefault` methods. Seems like removing them just adds a burden for writing geo scripts?